.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run daily at 06:45.
+# It will install Python dependencies and run tests with a variety of Python versions.
+
+name: Unit test smoke test
+
+on:
+  schedule:
+    - cron: 45 6 * * *
```

Comment:
Is this particular time meaningful, or just a random time in the morning?

Reply:
Random time in the morning. The github documentation recommends NOT running at the start of the hour, since a lot of folks pick that time. Midnight is also pretty popular. This seemed safe.